### DISCUSSION TITLE
Pace BMS - MODBUS

### DISCUSSION THREAD

**Discussion by markiospanero** [Louisvdw/dbus-serialbattery#570] [New BMS requests]
Please could you include the support for Pace BMS (V 1.5)
I think these are used in one of the other requests in the list so this may be of help there too.

I have uploaded the MODBUS addresses for the V1.5 which i have successfully used for other projects with these batteries.

Thanks

[PACE v1_5.pdf](https://github.com/Louisvdw/dbus-serialbattery/files/10211001/PACE.v1_5.pdf)

**Comment by SilverFox3333** [comment]
My older Pace BMS does not have CAN. But it does have RS485. I really want it to be able to communicate with my Victron Cerbo GX.
I also have (3) JK BMS's. I'd really like for Pace and JK to be in communication with my Cerbo GX at the same time! Is this possible?

**Comment by kalle-r** [comment]
https://www.akkudoktor.net/wp-content/uploads/wpforo/attachments/9832/72082=11511-pacers485-communication-protocol-pace-rs485-ms-20180615.pdf
